pkg/tagger/collectors/kubelet_extract.go

```diff
@@ -171,7 +171,7 @@ func (c *KubeletCollector) parsePods(pods []*kubelet.Pod) ([]*TagInfo, error) {
 			cLow, cOrch, cHigh := cTags.Compute()
 			entityID, err := kubelet.KubeContainerIDToTaggerEntityID(container.ID)
 			if err != nil {
-				log.Warnf("Unable to parse container: %s", err)
+				log.Warnf("Unable to parse container pName: %s / cName: %s / cId: %s / err: %s", pod.Metadata.Name, container.Name, container.ID, err)
```

Comment:
maybe we can only log the container name and ID here and drop the pod name

Reply:
I'd rather keep it. Let's say we have a log issue on PodX that has 50 replicas, we'll have the same container name, so I guess it eases the investigation if we have the pod name.